pvanalytics/quality/outliers.py

```diff
@@ -63,7 +63,8 @@ def zscore(data, zmax=1.5, nan_policy='raise'):
 
     """
     data = data.copy()
-    nan_mask = pd.Series([False] * len(data))
+    nan_mask = pd.Series([False] * len(data),
+                         index=data.index)
```

Comment:
Should this be considered a bugfix?  If so, it deserves a whatsnew note

Reply:
Ah, yes, good point. I'll add it to the whatsnew